api/api.pb.txt

```diff
@@ -2735,8 +2735,8 @@ file {
         label: LABEL_OPTIONAL
         type: TYPE_UINT32
         options {
-          65001: 0
           65003: "os.FileMode"
+          65001: 0
```

Comment:
All of these seemingly random changes came from running `DOCKER_SWARMKIT_USE_CONTAINER=1 make generate`. I'm not sure what the deal is there, so I split this part out to another commit to hopefully drop.